api/src/mutations/mint.rs

```diff
@@ -207,3 +309,13 @@ pub struct MintDropInput {
 pub struct MintEditionPayload {
     collection_mint: collection_mints::CollectionMint,
 }
+
+#[derive(Debug, Clone, InputObject)]
+pub struct RetryMintInput {
+    mint_id: Uuid,
```

Comment:
To align with other mutations like this to be id. When the mutation is scoped by an object like patch_drop or retry_mint make the object_id  named id.